hack/make-bundle-vars.mk

```diff
@@ -65,6 +65,11 @@ NOOBAA_BUNDLE_IMG_TAG ?= v5.9.0
 NOOBAA_BUNDLE_IMG_LOCATION ?= quay.io/noobaa
 NOOBAA_BUNDLE_IMG ?= $(NOOBAA_BUNDLE_IMG_LOCATION)/$(NOOBAA_BUNDLE_IMG_NAME):$(NOOBAA_BUNDLE_IMG_TAG)
 
+CSIADDONS_BUNDLE_IMG_TAG ?= v0.1.1
+CSIADDONS_BUNDLE_IMG_LOCATION ?= quay.io/csiaddons/k8s-bundle
+CSIADDONS_BUNDLE_IMG ?= $(CSIADDONS_BUNDLE_IMG_LOCATION):$(CSIADDONS_BUNDLE_IMG_TAG)
+CSIADDONS_BUNDLE_NAME ?= csi-addons
+
```

Comment:
Can you change it like below?

CSIADDONS_BUNDLE_NAME ?= csi-addons
CSIADDONS_BUNDLE_IMG_NAME ?= k8s-bundle
CSIADDONS_BUNDLE_IMG_TAG ?= v0.1.1
CSIADDONS_BUNDLE_IMG_LOCATION ?= quay.io/csiaddons
CSIADDONS_BUNDLE_IMG ?= $(CSIADDONS_BUNDLE_IMG_LOCATION)/$(CSIADDONS_BUNDLE_IMG_NAME):$(CSIADDONS_BUNDLE_IMG_TAG)